2018-05-23 19:55:36,562 - connectionpool.py[line:824] - DEBUG: Starting new HTTPS connection (1): api.siocloud.sioeye.cn
2018-05-23 19:55:36,562 - connectionpool.py[line:824] - DEBUG: Starting new HTTPS connection (1): api.siocloud.sioeye.cn
2018-05-23 19:55:37,751 - connectionpool.py[line:396] - DEBUG: https://api.siocloud.sioeye.cn:443 "POST /functions//login/ HTTP/1.1" 200 47
2018-05-23 19:55:37,751 - connectionpool.py[line:396] - DEBUG: https://api.siocloud.sioeye.cn:443 "POST /functions//login/ HTTP/1.1" 200 47
2018-05-23 19:55:37,753 - config_http.py[line:68] - DEBUG: "{\"username\": \"13688169291\", \"password\": \"e10adc3949ba59abbe56e057f20f883e\"}"
2018-05-23 19:55:37,753 - config_http.py[line:68] - DEBUG: "{\"username\": \"13688169291\", \"password\": \"e10adc3949ba59abbe56e057f20f883e\"}"
2018-05-23 19:55:37,754 - case.py[line:43] - DEBUG: {"status":100,"code":1,"msg":"app_id required"}
2018-05-23 19:55:37,754 - case.py[line:43] - DEBUG: {"status":100,"code":1,"msg":"app_id required"}
2018-05-23 19:55:37,780 - connectionpool.py[line:824] - DEBUG: Starting new HTTPS connection (1): api.siocloud.sioeye.cn
2018-05-23 19:55:37,780 - connectionpool.py[line:824] - DEBUG: Starting new HTTPS connection (1): api.siocloud.sioeye.cn
2018-05-23 19:55:37,818 - connectionpool.py[line:396] - DEBUG: https://api.siocloud.sioeye.cn:443 "POST /functions//register/ HTTP/1.1" 200 47
2018-05-23 19:55:37,818 - connectionpool.py[line:396] - DEBUG: https://api.siocloud.sioeye.cn:443 "POST /functions//register/ HTTP/1.1" 200 47
2018-05-23 19:55:37,820 - config_http.py[line:68] - DEBUG: "{\"nickname\": \"\\u5468\\u8e81\", \"password\": \"e10adc3949ba59abbe56e057f20f883e\", \"usertype\": \"\"}"
2018-05-23 19:55:37,820 - config_http.py[line:68] - DEBUG: "{\"nickname\": \"\\u5468\\u8e81\", \"password\": \"e10adc3949ba59abbe56e057f20f883e\", \"usertype\": \"\"}"
2018-05-23 19:55:37,820 - case.py[line:49] - DEBUG: 1
2018-05-23 19:55:37,820 - case.py[line:49] - DEBUG: 1
2018-05-23 19:55:37,820 - case.py[line:55] - DEBUG: 1
2018-05-23 19:55:37,820 - case.py[line:55] - DEBUG: 1
